Library/Homebrew/utils/github.rb

```diff
@@ -522,8 +524,42 @@ def self.fetch_pull_requests(name, tap_remote_repo, state: nil, version: nil)
     []
   end
 
+  # WARNING: This returns results in a slightly different form from `fetch_pull_requests`.
```

Comment:
Maybe worth clarifying what the form difference is here?

Reply:
TBH the exact difference isn't clear to me; all I did was compare the API response from looking up a PR using the search API vs the pulls API and noticed that they were different.

Currently we only really use the PR # from `fetch_pull_requests`, and both endpoints have this information.

Reply:
ah, I see, this might make it more obvious:

```suggestion
      # WARNING: The GitHub API returns results in a slightly different form here from `fetch_pull_requests`.
```